lib/IO/Framed/Read.pm

```diff
@@ -57,7 +57,7 @@ sub read {
 
         $bytes -= IO::SigGuard::sysread( $self->{'_in_fh'}, $self->{'_read_buffer'}, $bytes, $buf_len ) || do {
             if ($!) {
-                if ( !$!{'EAGAIN'} ) {
+                if ( !$!{'EAGAIN'} && !$!{'EWOULDBLOCK'}) {
```

Comment:
Just curious: does Win32/Perl give EWOULDBLOCK, or is this change in deference to certain *nix systems that give that error a different code from EAGAIN?